src/commands/language.ts

```diff
@@ -49,7 +50,8 @@ createSubcommand("language", {
   permissionLevels: [PermissionLevels.ADMIN],
   execute: async function (message, args) {
     //Old
-    const oldLanguageId = getCurrentLanguage(message.guildID) || 'en_US';
+    //@ts-ignore
```

Comment:
Here too.

Reply:
Same here

Reply:
This was fixed already. I do not know why it cannot be seen here. Can i resolve this?